flang/test/Parser/OpenMP/threadprivate-blank-common-block.f90

```diff
@@ -0,0 +1,9 @@
+! RUN: not %flang_fc1 -fsyntax-only %s -fopenmp 2>&1 | FileCheck %s
+! From Standard: A blank common block cannot appear in a threadprivate directive.
+
+program main
+    integer :: a
+    common//a
+    !CHECK: error: expected one of '$@ABCDEFGHIJKLMNOPQRSTUVWXYZ_'
```

Comment:
Would it be possible to issue a better error message?

Reply:
I'm afraid not, this error is coming from MessageExpectedText::ToString() (flang/lib/Parser.cpp) which is generic to all parser errors. Changing this might affect other tests. Please let me know if there is any better way to do it.